src/common/types.ts

```diff
@@ -25,8 +25,10 @@ export interface ILinterConfig {
     security: number
   }]
   securities?: ISecurities
-  offsetLine?: number
-  offsetColumn?: number
+  offsetLineStart?: number
+  offsetLineEnd?: number
```

Comment:
Please do not change the options names `offsetLine` and `offsetColumn`, it will breaking all exists users config witch using those options. we can just add the two new options.